README.md

```diff
@@ -28,85 +28,113 @@
-| [Automation](https://docs.mindsdb.com/mindsdb_sql/sql/create/jobs) | Automate AI-data workflows using Jobs | [(Python)](https://docs.mindsdb.com/sdks/python/create_job) | [(SQL)](https://docs.mindsdb.com/mindsdb_sql/sql/create/jobs) |
+## Connect Your Data
 
-## 💡 Example
+You can connect to hundreds of [data sources (learn more)](https://docs.mindsdb.com/integrations/data-overview). This is just an example of a Postgres database.
 
-### Connecting AI Agents to structured and unstructured data
-
-A common use case involves connecting agents to data. The following example shows how to connect an AI agent to a database so it can perform search over structured data:
-
-First we connect the datasource, in this case we connect a postgres database (you can do this via the SQL editor or SDK)
 ```sql
--- Step 1: Connect a data source to MindsDB
+
+-- Connect to demo postgres DB
 CREATE DATABASE demo_postgres_db
 WITH ENGINE = "postgres",
 PARAMETERS = {
-    "user": "demo_user",
-    "password": "demo_password",
-    "host": "samples.mindsdb.com",
-    "port": "5432",
-    "database": "demo",
-    "schema": "demo_data"
+  "user": "demo_user",
+  "password": "demo_password",
+  "host": "samples.mindsdb.com",
+  "port": "5432",
+  "database": "demo",
+  "schema": "demo_data"
 };
 
--- See some of the data in there
-SELECT * FROM demo_postgres_db.car_sales;
+```
+
+Once you've connected your data sources, you can [combine](https://docs.mindsdb.com/mindsdb_sql/sql/api/join-on), [slice it, dice it](https://docs.mindsdb.com/mindsdb_sql/sql/api/select), and [transform](https://docs.mindsdb.com/use-cases/data_enrichment/overview) it however your heart desires using good ol' standard SQL [(learn more)](https://docs.mindsdb.com/mindsdb_sql/sql/create/view). 
+
+After you've whipped your data into shape, it's time to build AI that actually learns!
+
+## Build AI Knowledge
+
+Our Knowledge Bases are state-of-the-art autonomous RAG systems that can digest data from any source MindsDB supports. Whether your data is structured and neater than a Swiss watch factory or unstructured and messy as a teenager's bedroom, our Knowledge Base engine will figure out how to find the relevant information. 
+
+
+**In this example** we will create a knowledge base that knows everything about amazon reviews. 
+
+
+```sql
+
+-- first you create a knowledge base
+CREATE KNOWLEDGE_BASE mindsdb.reviews_kb;
+
+
+-- now we can insert everything from the amazon reviews table into it, so it can learn it
+INSERT INTO mindsdb.reviews_kb (
+  SELECT review as content FROM demo_pg_db.amazon_reviews
+);
+
+-- You can check the status of your loads here
+SELECT * FROM information_schema.knowledge_bases;
 
```

Comment:
We can also add a command to show how to query the content of a knowledge base "SELECT * FROM mindsdb.reviews_kb".

Reply:
we can do that too, yes